packages/frontend-2/components/settings/workspaces/members/GuestsTable.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
Why not re-use MembersTable and just pass in a prop that does the minor adjustments that need to be made in the guests table (e.g. in the text labels)?

Reply:
For the most part its the same exact component

Reply:
@fabis94 It was I did originally, but it only exactly the same temporarily, when we decide on the exact content we will we need to separate it anyway, so I already separated it so next time we just need change/update the queries/table etc

Reply:
Is it really gonna be that different? I'd think that only the labels might change and maybe some of the actions. If you really think its gonna be different enough to require its own component, then we can merge this.

Reply:
Probably there will be different content in the table (like listening the parts this guest has access to), it's not very clear yet, if it's end up still being 99% the same I can still combine them :)